### ISSUE TITLE
Where to connect to a chat server? Tor hidden servers in Apache or Nginx?

### ISSUE THREAD

**Issue by bolzobrazil** [textbrowser/spot-on#19]
Hello,
 
where to connect to a public chat server? 

It could be helpful if users/clients can add an .onion-address and a Tor bootstrapper IP-address to get to a central Tor hidden server instance from the project side - or could the functionality be added to Apache or Nginx to work with this chat system?

https://community.torproject.org/onion-services/setup/

**Comment by textbrowser** [comment]
Spot-On provides free software. Web services are expensive and such requests ignore the purpose of Spot-On. You've already requested a similar paid-for service in Smoke.

https://github.com/textbrowser/smoke/issues/11

Create a project and offer people Web services. :) You cannot take people for granted by requesting that they also offer servers for free.

**TL;DR**

Spot-On does not include free servers.
